python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -1052,11 +1052,9 @@ def test_asset_node_is_executable(self, graphql_context: WorkspaceRequestContext
         assert result.data
         assert result.data["assetNodes"]
 
-        assert len(result.data["assetNodes"]) == 2
+        assert len(result.data["assetNodes"]) == 1
         exec_asset_node = result.data["assetNodes"][0]
         assert exec_asset_node["isExecutable"] is True
-        unexec_asset_node = result.data["assetNodes"][1]
-        assert unexec_asset_node["isExecutable"] is False
```

Comment:
The unexecutable external asset will no longer show up in the `assetNodes` for a job that uses it just for loading (this is equivalent to how source assets work)